.github/workflows/dockerimage.yml

```diff
@@ -28,17 +28,32 @@ jobs:
     - name: Build the Docker image
       run: docker build . --file Dockerfile --tag $(<IMAGE_ID)
 
-#     - name: Create github docker package
-#       env:
-#         GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # token provided by actions
-#       run: |
-#         # Authenticate
-#         echo "Authenticate with GITHUB_TOKEN"
-#         #docker login -u jesper-friis -p $GITHUB_TOKEN docker.pkg.github.com
-#         echo $GITHUB_TOKEN | docker login -u jesper-friis --password-stdin docker.pkg.github.com
-#         # Tag
-#         echo "Tag docker image: $(<IMAGE_ID) as $IMAGE_NAME:$(<VERSION)"
-#         docker tag $(<IMAGE_ID) docker.pkg.github.com/sintef/dlite/$IMAGE_NAME:$(<VERSION)
-#         # Publish
-#         echo "Publish docker image: docker.pkg.github.com/sintef/dlite/$IMAGE_NAME:$(<VERSION)"
-#         docker push docker.pkg.github.com/sintef/dlite/$IMAGE_NAME:$(<VERSION)
+    - name: Create github docker package
+      uses: whoan/docker-build-with-cache-action@v4
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}  # token provided by actions
+      with:
+        push_image_and_stages: false
+        username: jesper-friis
```

Comment:
Is possible to not hard code my username here or use sintef as username?
What happens if people other than me are pushing, will this workflow then fail or are the credential provided in the token used to publish using my username?

Reply:
I think we can ignore my comment above, since there must be a real user behind publishing a docker container. The workflow passes also when you are triggering it. Wonderful work Petter!